src/services/config.js

```diff
@@ -27,3 +27,8 @@ if (admin_pass) {
     pass: admin_pass
   }
 }
+
+config.auth_enabled = {
```

Comment:
Right, I just meant that in the code "config.auth_enabled" would look like a boolean, but I'm not sure if there is a better option. "config.auth.basic_enabled" is one other option. Not a big deal though, not trying to block the merge, just an idea.